src/api/index.test.js

```diff
@@ -17,135 +17,6 @@ import * as index from '.';
 
 beforeEach(jest.resetAllMocks);
 
-const serviceAccountPass = {
```

Comment:
A lot of tests being deleted here - what's left? What's testing the new unpatch logic?

Reply:
Yeah, the old approach was flawed and the tests centred around those methods which are now gone - there's nothing testing the new unpatch logic except from manual tests, I discussed this with @CarolynMabbott - what I've requested is for both unit tests and bash tests that would exercise this functionality

We're confident this fix is good but I've not had the time to write the tests owing to our upcoming deadline